### ISSUE TITLE
OpenRouter.ai Support

### ISSUE THREAD

**Issue by cvladan** [intellibar/main#69]
These days is much simpler to pay just one API as they're introducing latest model quite fast. So for a mere 5% of their commission, I started using them instead directly APIs. because I had registered and paid for five API providers and this way I just use one. They have both free models and paid ones.
 
So please add support for them. I think the API is the same as any other OpenAI compatible.

I also think it's easier for you, because you don't have to add new models all the time.
Thanks!

**Comment by cvladan** [comment]
Also, there is no need for proxy support if we use OpenRouter. I have already switched from using a proxy (in other applications) to using only OpenRouter to access the latest Google LLLM models, which are only available in the US.

**Comment by erusev** [comment]
Thanks for the suggestion! We'll look into it 👨‍💻

**Comment by juioudgdgeue894** [comment]
Any updates on this?

**Comment by astoilkov** [comment]
Yes, we now support openrouter.ai models or any custom model via the [custom models](https://docs.intellibar.app/using-custom-models) feature.
